### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [chrisstanarsenault/teacozy#1]
Great job linking all your images by using relative links!

One thing to consider on future projects is to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) for all your images. The alt attribute has a wide array of uses from improving the [accessibility](https://www.codecademy.com/articles/accessibility) of your page, to it being used as a backup, in case your image cannot be rendered. As well the alt attribute optimizes your site for search engine results. 

For example for this line of code you can do the following:
`<img src="./resources/images/img-donut.jpg" alt="Seasonal Donut on a plate" />`

https://github.com/chrisstanarsenault/teacozy/blob/ac5478189c634812fd2b1b0aeb17aa998ded59c5/index.html#L41
